test/e2e/autotls/auto_tls_test.go

```diff
@@ -47,7 +47,7 @@ import (
 // 1. Install cert-manager from `third_party/` directory.
 // 2. Run the command below to do the configuration:
 // kubectl apply -f test/config/autotls/certmanager/selfsigned/
```

Comment:
nit:
```suggestion
//    kubectl apply -f test/config/autotls/certmanager/selfsigned/
```
same like https://github.com/knative/serving/pull/6782/files#diff-462893639ceafde67ae16b6a81365509R93